packages/edmx-parser/utils/edmx.d.ts

```diff
@@ -345,14 +346,28 @@ declare namespace EDMX {
     export type UrlRefWrapper = Record<'UrlRef', URL>;
     export type RecordWrapper = Record<'Record', RecordExpression>;
     export type CollectionWrapper = Record<'Collection', CollectionExpression>;
-    export type ApplyWrapper = Record<'Apply', ApplyExpression>;
+    export type ApplyWrapper = {
+        Apply: ApplyExpression;
+        Function: ApplyExpression;
+    };
+    export type AndWrapper = Record<'And', ApplyExpression>;
```

Comment:
hmm, maybe these make sense, maybe not, but why is everything either an Apply or an If?

Reply:
They are just `any` in the end.
Not perfect but not improtant

Reply:
ok